homeassistant/components/camera/onvif.py

```diff
@@ -183,3 +196,23 @@ def handle_async_mjpeg_stream(self, request):
     def name(self):
         """Return the name of this camera."""
         return self._name
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attrs = super().device_state_attributes
+        custom_attrs = {}
+        i = 0
+        for profile in self._profiles:
+            bounds = profile.VideoSourceConfiguration.Bounds.__dict__
+            custom_attrs.update({"Profile #" + str(i):
```

Comment:
About that __dict__ thing. That is pretty complicated, as profile object is a result of unmarschalling wsdl service response by suds library. suds has a naming convention that if a field to be unmarschalled is marked as required, it is unmarschalled with "_" prefix. That causes a lot of confusion with python naming convention, but that's what they did in suds. I wanted to change suds library, but it is not maintained since few years, so there's no easy way to change this.
I tried to do "._width", but it caused lint error. To fix it, I changed it to "__dict__["_width"]".

Reply:
Disable the linter instead